docs/index.md

```diff
@@ -115,12 +115,47 @@ The APIs are organized as follows:
             </dd>
         </dl>
     </li>
-    <li>
+    <li class="bigger ground-layer">
         <dl>
-            <dt>
-                <a title="Pentaho Web Package" href="platform/web-package">Web Package</a>
+            <dt id="ground">
+                Pentaho Web Platform
             </dt>
-            <dd>The deployment, dependency and versioning unit for web resources.</dd>
+            <dd>
+                <ul>
+                    <li>
+                        <dl>
+                            <dt>
+                                <a title="Pentaho Web Package" href="platform/web-package">Pentaho Web Package</a>
+                            </dt>
+                            <dd>The dependency and versioning unit for JavaScript web resources
+                                in the Pentaho platform.</dd>
+                        </dl>
+                    </li>
+                </ul>
+                <ul>
+                    <li>
+                        <dl>
+                            <dt>
+                                <a title="OSGi Bundle" href="platform/osgi-web-project">OSGi Web Project</a>
```

Comment:
This introduces a new term/concept. We're also getting "OSGi" all over the place, that shouldn't be needed.

I think this page can be at the platform level but linked inside the Pentaho Web Package page.